Project-II-Calculator/calculator/src/components/ButtonComponents/NumberButton.js

```diff
@@ -1,2 +1,26 @@
 import React from 'react';
 import './Button.css';
+
+// - Create a `<NumberButton />` component that can accept `props` and display any number/symbol passed down as `text`.
+//   - Example your component should be able to render a dynamic prop called text:
+//   - Your button button should also be able to accept dynamic props `buttonStyle` for styling
+
+function NumberButton(props) {
+    <div className= {props.NumberBtns.buttonStyle}>{props.NumberBtns.text}</div>
+}
+
+
+const NumberBtns = [
```

Comment:
Similar to the *ActionButton*, this *numberBtns* can be placed inside the *NumberButton* component.